view/frontend/templates/js/replacejs.phtml

```diff
@@ -1514,6 +1514,8 @@ if (!$block->isSaveCartInSections()) { ?>
                 if (callbacks.success_url) {
                     // redirect on success order save
                     location.href = callbacks.success_url;
+                    // invalidate bolt cart to prevent bolt initialization on thank you page
+                    customerData.reload(['boltcart'], true);
```

Comment:
The **close** callback is not called In pre-auth flow. At least it wasn't for many months. But perhaps it is now. Can you verify?

Reply:
Sure. Do you mean bolt server didn't call **close** callback for pre-auth flow? Or it was a reason in our plugin?

Reply:
The execution flow is taken over by Bolt checkout (connect.js). In pre-auth the success redirect URL is passed on in create order call and sent back to checkout modal in the successful authorize response data. In that case, when the final `callback();` passed to the success checkout callback is called Bolt checkout (modal, connect.js) does not trigger the close callback but proceeds to redirect. You can not rely on this here, it does not happen.